config/environments/test.rb

```diff
@@ -37,10 +37,12 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  #config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
+  config.action_mailer.default_url_options = {host: "localhost", port: 3000, protocol: 'http' }
```

Comment:
@nagerz  I think I have  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 } and not the other one. Once we merge, let's test and see what we actually need.